### ISSUE TITLE
Vbrowser Argument Syntax

### ISSUE THREAD

**Issue by xanzadu** [salimk/Rcrawler#54]
Line 516 of the code:

"cat("browser:"+i+" port:"+pkg.env$Lbrowsers[[i]]$process$port)" 

returns non-numeric argument to binary operator error. 

It should use "," instead of "+".  I.e.

"cat("browser:",i," port:",pkg.env$Lbrowsers[[i]]$process$port)"

**Comment by tomwwagstaff-product** [comment]
Also experiencing this error (but didn't know why!)

**Comment by KnutJaegersberg** [comment]
yes, this error keeps bugging me too. what i ended up doing was downloading the code, changing the error and rezipping it to make it installable without the error. 
important error! without the javascript rendering via browser is not usable.

**Comment by systemnova** [comment]
I'm getting the same error, but this code seems to have already been updated to the change proposed above.
Are there any other trouble shooting steps to get Vbroswer working when it produces the following error:

> Preparing browser process Error in "browser:" + i : non-numeric argument to binary operator

**Comment by MartinsRodrigo** [comment]
I´m also getting this error. Any solution?
